### PR TITLE
Updated README for the Social Bias Frames dataset

### PR DESCRIPTION
**Pull request by mcmillanmajora**
See the updated card at https://github.com/mcmillanmajora/datasets/tree/add-SBIC-card/datasets/social_bias_frames. I incorporated information from the [SBIC data statement](https://homes.cs.washington.edu/~msap/social-bias-frames/DATASTATEMENT.html), paper, and the corpus README file included with the dataset download.

### CODE REVIEW

File: datasets/social_bias_frames/README.md [hunk 1]
```diff
@@ -29,33 +49,38 @@
 ## [Dataset Description](#dataset-description)
 
 - **Homepage:** [https://homes.cs.washington.edu/~msap/social-bias-frames/](https://homes.cs.washington.edu/~msap/social-bias-frames/)
-- **Repository:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
-- **Paper:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
-- **Point of Contact:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+- **Repository:** [https://homes.cs.washington.edu/~msap/social-bias-frames/](https://homes.cs.washington.edu/~msap/social-bias-frames/)
+- **Paper:** [Social Bias Frames: Reasoning about Social and Power Implications of Language](https://www.aclweb.org/anthology/2020.acl-main.486.pdf)
+- **Leaderboard:**
+- **Point of Contact:** Maartin Sap msap@cs.washington.edu
```
Comment (yjernite) [inline datasets/social_bias_frames/README.md:55]:
```suggestion
- **Point of Contact:** [Maartin Sap](mailto:msap@cs.washington.edu)
```

**Review by yjernite — APPROVED** [review]
Looks good!

File: datasets/social_bias_frames/README.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+- found 
+languages: 
+- en 
+licenses: 
+- cc-by-4.0 
+multilinguality: 
+- monolingual 
+size_categories: 
+- 100K<n<1M 
+source_datasets: []
```
Comment (lhoestq) [inline datasets/social_bias_frames/README.md:14]:
```suggestion
source_datasets:
- original
```

**Review by lhoestq — APPROVED** [review]
Nice thank you :)

**Pull request merged**